docs/high-availability/Vagrantfile

```diff
@@ -0,0 +1,51 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  fedora = 'hansode/fedora-21-server-x86_64'
+  script = "provision.sh"
+  #discovery_url = `wget https://discovery.etcd.io/new?size=3 -O -`
+  discovery_url = "dummy"
```

Comment:
I see your point.  I'd like to comment it out (because there is some value in keeping the vagrant structure of getting the unique discovery url before provisioning, even though the recipe will not need to use it for a single node default setup.  The idea is that users can make the minor modification to support clustered kubeletized etcd.